src/app/nation/nation.component.html

```diff
@@ -11,6 +11,7 @@ <h1 i18n>Nation List</h1>
           <th i18n>Nation Name</th>
           <th i18n>Admin Name</th>
           <th i18n>Nation Url</th>
+          <th i18n>Community List</th>
```

Comment:
This one can come along with Action. No need to separate `<td>` just for a button